.github/workflows/ci.yml

```diff
@@ -17,8 +17,20 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
+        pip install --upgrade pip setuptools wheel
         pip install --upgrade -r requirements.txt
-        pip install .
+    - name: Build package
+      run: |
+        python ./setup.py bdist_wheel
+        mv ./dist/*.whl ./dist/netaddr-0.CI-py2.py3-none-any.whl
+    - name: Upload the wheel artifact
+      uses: actions/upload-artifact@v2
```

Comment:
Uploading artifacts is out of scope of this PR I'd argue.

Reply:
uploaded artifacts are nice for debugging, also allow people to use your wheels without building them themselves (though it is discouraged ). But yes, they are out of scope of this PR.